src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: calc((#{$cell-size} * #{$columns}) +
+  (#{$padding-horizontal} * 2) + (#{$gap} * #{$columns}));
+  padding: $padding-vertical $padding-horizontal;
+
+  @each $day , $value in $days {
+    &--start--#{$day} &__day:nth-child(1) {
+      margin-left: $cell-size * $value + ($gap * $value);
+    }
+  }
+
+  @for $i from 28 through 31 {
```

Comment:
Would be better if you would write the variable more correctly. For example `$day` as you did above in the prev loop